<ui version="4.0" >
 <class>BranchUi</class>
 <widget class="QDialog" name="BranchUi" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>219</width>
    <height>99</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="3" >
    <layout class="QVBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QLabel" name="branchLA" >
       <property name="text" >
        <string>&amp;Available branches:</string>
       </property>
       <property name="buddy" >
        <cstring>branchCO</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="branchCO" >
       <property name="toolTip" >
        <string>Select your branch</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>31</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2" >
    <widget class="QPushButton" name="closePB" >
     <property name="text" >
      <string>&amp;Close</string>
     </property>
     <property name="default" >
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QPushButton" name="okPB" >
     <property name="text" >
      <string>&amp;OK</string>
     </property>
     <property name="default" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>branchCO</tabstop>
  <tabstop>okPB</tabstop>
  <tabstop>closePB</tabstop>
 </tabstops>
 <includes>
  <include location="local" >qt_i18n.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
